source/source.json

```diff
@@ -6110,11 +6112,299 @@
+				}
+			]
+		}
+	},
+	{
+		"github": "M4nueleZ/Luma3DS_ITA",
+		"title": "Luma 3DS Ed. Italiana",
+		"systems": ["3DS"],
+		"categories": ["firm", "luma3ds", "translation"],
+		"long_description": "Questa è l'edizione italiana di Luma3DS, un noob-proof custom firmware per console della famiglia Nintendo 3DS.",
+		"icon": "https://raw.githubusercontent.com/M4nueleZ/Luma3DS_ITA/master/luma3dslogoita.png",
+		"archive": {
+			"Luma3DS_ITA.*\\.zip": {
+				"boot.firm": ["boot.firm"]
+			}
+		}
+	},
+	{
+		"title": "TasmanQuest",
+		"systems": ["3DS"],
+		"categories": ["game"],
+		"author": "Nawias",
+		"description": "Join Taśman in his quest to wrap everything in TAŚMAN Sealed duct tape!",
+		"long_description": "In this short, arcade-y game for your Nintendo 3DS explore the famous Lewiatan grocery store of Księżomierz and flex your tape-wrapping skills as the clock is ticking!\n\nYou play as Adam a.k.a. \"Taśman\". You came to the local Lewiatan of Księżomierz to promote your brand - TAŚMAN™ Sealed. But it's Kasia's shift today and she won't allow it! Wrap as many items in your duct tape and get back in Zakre's Seiac before the guards catch you!\n### Credits\nMost of the game assets come from [LimeZu's](https://limezu.itch.io/) packs: Modern Interiors, Modern Exteriors, Modern UI, Serene Village Revamped\n\n\nMIDI files used for soundtrack come from MuseScore:\n\nMenu Music - [Loch Lomond](https://musescore.com/user/24691996/scores/6908959)\n\nAction Music - [Break the Rules](https://musescore.com/user/15821421/scores/6878141)\n\nNintendo 3DS port of LÖVE - [LÖVE Potion](https://github.com/lovebrew/lovepotion) by TurtleP",
+		"website": "https://nawias.itch.io/tasmanquest",
+		"download_page": "https://nawias.itch.io/tasmanquest",
+		"image": "http://nawiasdev.eu/tasmanquest/logo.png",
+		"icon": "http://nawiasdev.eu/tasmanquest/icon.png",
+		"created": "2023-09-01T00:00:00Z",
+		"updated": "2023-09-13T00:00:00Z",
+		"version": "1.0.0",
+		"version_title": "Full Release",
+		"downloads": {
+			"TasmanQuest.3dsx": {
+				"url": "http://nawiasdev.eu/tasmanquest/TasmanQuest.3dsx"
+			}
+		}
+	},
+	{
+		"github": "Alexyo21/the-pirate-launcher",
+		"title": "Pirate launcher",
+		"systems": ["3DS"],
+		"categories": ["utility"],
+		"description": "If you like pirating games and so on you can’t miss this opportunity.💀❄️",
+		"long_description": "Refurbished build of the pirate launcher, with some new debug feature and up to date to the latest updates from official source, it doesn’t do nothing, only proves you are a pirate💀",
+		"icon": "https://raw.githubusercontent.com/Alexyo21/the-pirate-launcher/master/icon.png",
+		"scripts": {
+			"Install the Pirate Launcher": [
+				{
+    					"type": "promptMessage",
+    					"message": "This will replace the boot.3dsx in the root of\nyour SD card, effectively replacing the Homebrew\nLauncher by the Pirate Launcher upon launching the app.\nContinue ?"
```

Comment:
It is needed, since people might wonder what the app does. Some might even think like me, like it would install a new cia or something but no. So i feel like it's needed to be a bit explicit of what's happening